modules/wizard/view-wizard-step-media.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Displays the wizard media step
+ *
+ * @since 2.7
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Don't access directly.
+};
+
+$default_options = PLL_Install::get_default_options();
+$options = wp_parse_args( get_option( 'polylang' ), $default_options );
+$media_support = $options['media_support'];
+$rtl = '';
+if ( function_exists( 'is_rtl' ) && is_rtl() ) {
+	$rtl = '-rtl';
+}
```

Comment:
Why `is_rtl` would not exist?  I see none.
`$rtl = is_rtl() ? '-rtl' : '';`
should be sufficient.

Reply:
function_exists( 'is_rtl' ) inspired by WP code seen somewhere.
I could have used ternary operator instead :)
Corrected